### ISSUE TITLE
Unable to enable build/run actions delegation to Gradle

### ISSUE THREAD

**Issue by dtsaryov** [JetBrains/gradle-idea-ext-plugin#28]
Environment:
- IJ IDEA CE 2018.1.6
- Plugin version: 0.2

Desirable configuration:
```Groovy
buildscript {
    repositories {
        maven {
            url "https://plugins.gradle.org/m2/"
        }
    }
    dependencies {
        classpath group: 'gradle.plugin.org.jetbrains.gradle.plugin.idea-ext', name: 'gradle-idea-ext', version: 0.2
    }
}

apply(plugin: 'org.jetbrains.gradle.plugin.idea-ext')

idea.project.settings {
    delegateActions {
        delegateBuildRunToGradle = true
        testRunner = 'GRADLE'
    }
}
```

But no effect found

![image](https://user-images.githubusercontent.com/6183810/43580229-bc258102-9665-11e8-9be6-5d983a32c4c9.png)

Test project:

[builddelegation.zip](https://github.com/JetBrains/gradle-idea-ext-plugin/files/2253227/builddelegation.zip)

**Comment by nskvortsov** [comment]
Unfortunately, you need IDEA 2018.2+ for it to work.

**Comment by dtsaryov** [comment]
@nskvortsov thank you for explanation. I've just missed IJ IDEA requirements in README.

**Comment by admizh** [comment]
@nskvortsov "delegate IDE build" still not for for me. plugin actions not affect to this settings:

![image](https://user-images.githubusercontent.com/6014483/43826953-aa373f78-9b09-11e8-8df5-0ff7c91e194a.png)

my snippet (with plugin gradle-idea-ext:0.4.1):
```
idea.project.settings {
    compiler {
        processHeapSize = 5000
        parallelCompilation = true
        displayNotificationPopup = true
        autoShowFirstErrorInEditor = true
    }
    delegateActions {
        delegateBuildRunToGradle = true
        testRunner = ActionDelegationConfig.TestRunner.GRADLE
    }
    runConfigurations {
        defaults(TestNG) {
            vmParameters = "-ea -javaagent:${javaAgentArg}"
        }
    }
}
```

**Comment by dtsaryov** [comment]
@admizh have you tried to use a String as `testRunner` property value? And check that your project is imported as Gradle project. The feature successfully works for me.

**Comment by admizh** [comment]
@dtsaryov 
1. No i use 'testRunner' as ENUM, also my project was imported as 'gradle'. can help to debug...?

**Comment by dtsaryov** [comment]
@admizh could you try to reproduce the problem with my test project - `builddelegation.zip`?

**Comment by admizh** [comment]
@dtsaryov yes, i tried your project - not working

Also, i tried to change plugin version to 0.4.1, and set flag "use gradle 'wrapper' task configuration" on import.
![image](https://user-images.githubusercontent.com/6014483/43832014-0fe3519c-9b17-11e8-8db8-c055be047f07.png)

Below (finall snippet): 
```
buildscript {
    repositories {
        maven {
            url "https://plugins.gradle.org/m2/"
        }
    }
    dependencies {
        classpath group: 'gradle.plugin.org.jetbrains.gradle.plugin.idea-ext', name: 'gradle-idea-ext', version: '0.4.1'
    }
}

apply(plugin: 'org.jetbrains.gradle.plugin.idea-ext')

wrapper {
    gradleVersion = '4.9'
    distributionType = org.gradle.api.tasks.wrapper.Wrapper.DistributionType.ALL
}

idea.project.settings {
    delegateActions {
        delegateBuildRunToGradle = true
        testRunner = 'GRADLE'
    }
}

group 'ideaplugintest'
version '1.0-SNAPSHOT'
```

@nskvortsov  my IDEA 2018.2.1 was installed via update mechanism (from menu "Help --> Check for updates") via PATCH , may be some old idea configs was migrated from previous idea, and it affected to plugin work?

**Comment by nskvortsov** [comment]
Fixes will be delivered in 2018.2.x

**Comment by nskvortsov** [comment]
delivered in 2018.2.3
